scripts/gha/integration_testing/xcodebuild.py

```diff
@@ -58,18 +53,17 @@ def get_args_for_archive(
     Sequence of strings, corresponding to valid args for a subprocess call.
   """
   xcode_args = get_args_for_build(
-      path, scheme, output_dir, ios_sdk, dev_team, configuration)
+      path, scheme, output_dir, ios_sdk, configuration)
   additional_device_args = [
-      "PROVISIONING_PROFILE=" + uuid,
-      "CODE_SIGN_STYLE=Manual",
-      "-archivePath", archive_path,
-      "archive"
+      "CODE_SIGN_IDENTITY=\"\"",
```

Comment:
But it will looks inconsistent with other args.

Reply:
More readable, though. PEP8 explicitly prescribes doing this to avoid escape characters in the string:
https://www.python.org/dev/peps/pep-0008/#string-quotes